tests/phpunit/CRM/Utils/TokenConsistencyTest.php

```diff
@@ -105,16 +105,17 @@ public function testCaseTokenConsistency(): void {
    * @return string
    */
   protected function getExpectedCaseTokenOutput(): string {
-    return '1
-Housing Support
-Case Subject
-July 23rd, 2021
-July 26th, 2021
-case details
-Ongoing
-No
-' . CRM_Utils_Date::customFormat($this->case['created_date']) . '
-' . CRM_Utils_Date::customFormat($this->case['modified_date']) . '
+    return 'case.id :1
+case.case_type_id:label :Housing Support
+case.subject :Case Subject
```

Comment:
The colon spacing seems odd but not a blocker. I can see that if the token contains a colon it would look just as weird to have it stuck to the left side. Maybe some other character? e.g. `case.case_type_id:label | Housing Support`

Reply:
@demeritcowboy no - you have no idea how much pain it took to get to this point - please don't make me change it

Reply:
also fyi I originaly had a space either side of the `:` - what happened was the IDE kept stripping it & I spent hours staring at strings to figure out how they were different

Reply:
thanks!